addons/binding/org.openhab.binding.opensprinkler/ESH-INF/i18n/opensprinkler_en_US.properties

```diff
@@ -0,0 +1,208 @@
+# binding

```

Comment:
Do we need this file? I.e. are all texts different in en_US than in the default en?